betelgeuse/parser.py

```diff
@@ -187,3 +187,30 @@ def parse_docstring(docstring=None):
             field_value = output
             fields_dict[field_name] = field_value
     return fields_dict
+
+
+def parse_markers(all_markers=None):
+    """Parse the markers."""
+    ignore_list = ['parametrize', 'skipif', 'usefixtures', 'skip_if_not_set']
```

Comment:
some of these seem to be `robottelo`-specific.
Also, since you wish to ignore `skipif` marker, I'd say you'd like to also ignore `rhel_ver_match`, one etc.

My point here is - perhaps the `ignore_list` might be implemented as a CLI option rather than a static, hardcoded list?

It would make it much more flexible and easy to add more over time.

Reply:
@rplevka Your wish is granted - Now the ignore list is configurable through `betelgeuse_config` module https://github.com/SatelliteQE/betelgeuse/pull/141/commits/7b15cbf21ca4e1329b75de34d2d0457f2ef55699